wdl/EvidenceQC.wdl

```diff
@@ -169,5 +196,83 @@ workflow EvidenceQC {
     File bincov_matrix = MakeBincovMatrix.merged_bincov
     File bincov_matrix_index = MakeBincovMatrix.merged_bincov_idx
     File bincov_median = MedianCov.medianCov
+
+    File? qc_table_csv = MakeQcTable.qc_table_csv
+  }
+}
+
+task MakeQcTable {
+  input {
+
+    File ploidy_plots
+    File bincov_median
+    File WGD_scores
+    Array[Float]? melt_insert_size
+    Array[String] samples
+
+
+    File? manta_qc_low
+    File? manta_qc_high
+    File? melt_qc_low
+    File? melt_qc_high
+    File? wham_qc_low
+    File? wham_qc_high
+    File? scramble_qc_low
+    File? scramble_qc_high
+
+    String sv_pipeline_docker
+    String output_prefix
+    RuntimeAttr? runtime_override
   }
+
+  output {
+    File qc_table_csv = "${output_prefix}.evidence_qc_table.tsv"
+  }
+
+  command <<<
+    set -euxo pipefail
+
+    if ~{defined(melt_insert_size)} ; then
+      echo -e "sample_ID\tmean_insert_size" > mean_insert_size.tsv
+      mv ~{if (defined(melt_insert_size)) then write_tsv(transpose([samples, select_first([melt_insert_size])])) else ""} mean_insert_size.tsv.tmp
+      cat mean_insert_size.tsv.tmp >> mean_insert_size.tsv
+    fi
+
+    tar -xvf ~{ploidy_plots}
+
+    python /opt/sv-pipeline/scripts/make_evidence_qc_table.py \
+      ~{"--estimated-copy-number-filename " + "./ploidy_est/estimated_copy_numbers.txt.gz"} \
+      ~{"--median-cov-filename " + bincov_median} \
+      ~{"--wgd-scores-filename " + WGD_scores} \
+      ~{"--binwise-cnv-qvalues-filename " + "./ploidy_est/binwise_CNV_qValues.bed.gz"} \
+      ~{"--manta-qc-outlier-high-filename " + manta_qc_high} \
+      ~{"--melt-qc-outlier-high-filename " + melt_qc_high} \
+      ~{"--wham-qc-outlier-high-filename " + wham_qc_high} \
+      ~{"--manta-qc-outlier-low-filename " + manta_qc_low} \
+      ~{"--melt-qc-outlier-low-filename " + melt_qc_low} \
+      ~{"--wham-qc-outlier-low-filename " + wham_qc_low} \
+      ~{if (defined(melt_insert_size)) then "--melt-insert-size " + "mean_insert_size.tsv" else ""}\
+      ~{"--output-prefix " + output_prefix}
+  >>>
+#
+  RuntimeAttr runtime_default = object {
+    cpu_cores: 1,
+    mem_gb: 4,
+    boot_disk_gb: 10,
+    preemptible_tries: 3,
+    max_retries: 1,
+    disk_gb: 100
```

Comment:
Not required before merge, but should come back to this: Should do resource optimization to slim down the default memory and disk